ThreeBotPackages/tfgrid/threebot_provisioning/chatflows/threebot_reservation.py

```diff
@@ -0,0 +1,135 @@
+from Jumpscale import j
+import netaddr
+import random
+import nacl
+import os
+
+
+def chat(bot):
+
+    """
+    This chat is create 3bot container using zosv2
+    """
+    explorer = j.clients.threebot.explorer
+    cl = j.clients.s3.get("deployer")
+
+    AWS_ID = cl.accesskey_
+    AWS_SECRET = cl.secretkey_
+
+    user_info = bot.user_info()
+    name = user_info["username"]
+    email = user_info["email"]
+    ips = ["IPV6", "IPV4"]
+    choose = ["New", "Restore"]
+    ip_range_choose = ["Specify IP Range", "Choose IP Range for me"]
```

Comment:
please put a header first saying:
"to have access to the threebot, the network must be configured"


and the choices are:

"Configure IP range myself"

"Choose IP range for me"